openedx/features/course_experience/static/course_experience/js/CourseSock.js

```diff
@@ -55,6 +55,10 @@ export class CourseSock {  // eslint-disable-line import/prefer-default-export
       $toggleActionButton.toggleClass('active').toggleClass('aria-expanded');
       $verificationSock.slideToggle(toggleSpeed, fixUpgradeButton);
 
+      // Toggle aria-expanded attribute
+      const newAriaExpandedState = !!$toggleActionButton.attr('aria-expanded') ? 'false' : 'true';
```

Comment:
Could this just be 
const newAriaExpandedState = !$toggleActionButton.attr('aria-expanded')